packages/react-app-revamp/components/_pages/WizardFormCreateContest/Step3/schema.ts

```diff
@@ -24,6 +26,10 @@ export const schema = object({
   contestDescription: string()
     .trim()
     .min(1),
+  useSameTokenForSubmissions: boolean(),
+  submissionTokenAddress: string()
+    .regex(/^0x[a-fA-F0-9]{40}$/)
+    .or(string().max(0)),
```

Comment:
How come we are doing this differently than the `votingTokenAddress` below?

Reply:
The voting token address is always required ; meanwhile the submission token address depends on the value of `useSameTokenForSubmissions`: if `true` we can leave it empty (hence `or(string().max(0))`, otherwise, it needs to be an ethereum address

Reply:
I think this may be causing me not to be allowed to edit it except for by having it imported by minting my own, I reference this bug [here](https://github.com/JokeDAO/JokeDaoV2Dev/pull/68#discussion_r952980873).

Reply:
Was caused by something else, I pushed the fix :heavy_check_mark: